### PR TITLE
smooth brain no wrinkle

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

image
image

whenever you wash a brain in a sink or in a washing machine, it now changes to a sprite of a proper smooth brain, with minimal wrinkles and fast thinking action. hurrah.

## Why It's Good For The Game

smooth brain = faster thinking

## Changelog

:cl:
add: smooth brains
/:cl:

### CODE REVIEW

File: code/modules/mob/living/brain/brain_item.dm [hunk 1]
```diff
@@ -316,6 +316,8 @@
 	else
 		setOrganDamage(BRAIN_DAMAGE_DEATH)
 
+	icon_state = "brain-w"
```
Comment (Iamgoofball) [inline code/modules/mob/living/brain/brain_item.dm:319]: make sure that brain surgery fixes this

**Review by Imaginos16 — APPROVED** [review]
Excellent spritework

**Comment by tralezab** [comment]
please get some gbp first, you're in the negative here. i do like the sprite, but i don't like the achievability of it being expanded to any sink.

File: code/modules/mob/living/brain/brain_item.dm (hunk 1 above)
Comment (magatsuchi) [reply code/modules/mob/living/brain/brain_item.dm:319]: will do asap

**Pull request closed without merging**